model/sys/vmem_ptw.sail

```diff
@@ -37,15 +35,13 @@ function ptw_error_to_str(e : PTW_Error) -> string = {
   }
 }
 
-// PUBLIC
 overload to_str = {ptw_error_to_str}
```

Comment:
This one seems like it was originally wrong? `union PTW_Error` is private and `ptw_error_to_str` is also private so